### DISCUSSION TITLE
/diversity/comment

### DISCUSSION THREAD

**Discussion by huazie** [huazie/hexo-theme-diversity#12] [Announcements]
# Hello Diversity
博客多样性，一款多主题自由切换的Hexo主题
项目地址：[hexo-theme-diversity](https://github.com/huazie/hexo-theme-diversity)

**Comment by laorui** [comment]
我很喜欢此主题。

**Comment by laorui** [comment]
有谁能告诉我，怎么使用它？

**Reply by huazie** [reply]
可以参考我这里的文章，有啥不明白的[过来留言](https://blog.huazie.com/diversity/comment/)哈
https://blog.huazie.com/diversity/blog/?path=/categories/%E5%8D%9A%E5%AE%A2%E6%A1%86%E6%9E%B6-Hexo/

**Comment by huazie** [comment]
## Hello Diversity
博客多样性，一款多主题自由切换的 Hexo 主题

项目地址：[hexo-theme-diversity](https://github.com/huazie/hexo-theme-diversity)
